src/index.css

```
@@ -156,4 +156,204 @@ h1 {
+@keyframes float {
+  0%, 100% {
+    transform: translateY(0) translateX(0);
+    opacity: 0;
+  }
+  10% {
+    opacity: 1;
+  }
+  90% {
+    opacity: 1;
+  }
+  100% {
```

Stylelint (17.2.0)

[error] Unexpected duplicate "100%" (keyframe-block-no-duplicate-selectors)

(keyframe-block-no-duplicate-selectors)